### ISSUE TITLE
TLS Error: unable to get local issuer certificate

### ISSUE THREAD

**Issue by mike10010100** [habitat-sh/habitat#7777]
OS: Ubuntu 18.04
Hab Version 1.6.56/20200618202635
HAB environment variables:
AB_AUTH_TOKEN:<token>
HAB_BLDR_URL: <on-prem-builder>

Any hab build errors out on download of dependent packages

```
2020-06-30T10:41:42.5345930Z    protobuf-c: Plan loaded
2020-06-30T10:41:42.5347885Z    protobuf-c: Validating plan metadata
2020-06-30T10:41:42.5602479Z    protobuf-c: Using HAB_BIN=/hab/pkgs/core/hab/1.6.56/20200618202635/bin/hab for installs, signing, and hashing
2020-06-30T10:41:42.5604740Z    protobuf-c: hab-plan-build setup
2020-06-30T10:41:42.5642208Z    protobuf-c: Writing pre_build file
2020-06-30T10:41:42.5810844Z    protobuf-c: Resolving build dependencies
2020-06-30T10:41:42.5901048Z [0m[0m[1m[33mÂ» Installing core/gcc[0m
2020-06-30T10:41:42.5903458Z [0m[0m[1m[32mâ˜ Determining[0m latest version of core/gcc in the 'stable' channel
2020-06-30T10:41:42.8297866Z [0m[0m[1m[31mâœ—âœ—âœ—[0m
2020-06-30T10:41:42.8303131Z [0m[0m[1m[31mâœ—âœ—âœ— error sending request for url (https://<on-prem-depot>/v1/depot/channels/core/stable/pkgs/gcc/latest?target=x86_64-linux): error trying to connect: error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed:ssl/statem/statem_clnt.c:1913: (unable to get local issuer certificate)[0m
2020-06-30T10:41:42.8306938Z [0m[0m[1m[31mâœ—âœ—âœ—[0m
2020-06-30T10:41:42.8452125Z    protobuf-c: WARN: Could not find a suitable installed package for 'core/gcc'
2020-06-30T10:41:42.8485401Z    protobuf-c: ERROR: Resolving 'core/gcc' failed, should this be built first?
2020-06-30T10:41:42.8537720Z    protobuf-c: Build time: 0m0s
2020-06-30T10:41:42.8539584Z    protobuf-c: Exiting on error
2020-06-30T10:41:42.8669930Z Script done, file is /src/results/logs/protobuf-c.2020-06-30-104142.log
2020-06-30T10:41:43.1276274Z Build failed.
```

The only solution is to manually set this environment variable:

`SSL_CERT_FILE: /etc//ssl/certs/ca-certificates.crt`

It would seem there's some kind of regression happening, since 1.6.0 was fine.

**Comment by stevendanna** [comment]
@mike10010100 Thanks for filing this. I recently hit the same thing. Definitely a regresssion.  @davidMcneil has done a bit of digging but is still investigating.

**Comment by davidMcneil** [comment]
Thanks for opening this. It is resolved by  #7778.
